dev-requirements.txt

```diff
@@ -7,4 +7,5 @@ codecov
 rst2pdf
 Sphinx
 mock==2.0.0
-codacy-coverage
\ No newline at end of file
+codacy-coverage
+PyQt5
```

Comment:
Looks like PyQt5 package doesn't exist (not maintained?) for Python2.7. It's possible to add conditional record to requirements file. It's easy to google. Also if you don't use this app yet, you may not add PyQt5 to requirements right now.

Reply:
https://stackoverflow.com/q/19559247/3648361